src/rocksdb/src/lib.rs

```diff
@@ -244,48 +249,55 @@ where
         let dynamic_config = tuning_config.dynamic.clone();
         if options.cleanup_on_new && instance_path.exists() {
```

Comment:
Why is `cleanup_on_new` an option? It seems like it's true always?

Reply:
I had this very early on and just never removed it! I can cleanup in a followup